articles/tables.md

```diff
@@ -388,17 +394,17 @@ schema = syn.store(schema)
 {% tab R %}
 {% highlight r %}
 # Get the latest table
-table <- synGet('syn7266590')
+schema <- synGet('syn7266590')
 # delete the 'purchased' column
-schema <- synRemoveColumn(table, table@columns[[5]])
+schema <- schema$removeColumn(newColumn)
```

Comment:
why isn't this `synDelete(newColumn)`?

Reply:
ignore - columns to delete are tied to the schema they are in. can't universally delete a column.